api/core/workflow/nodes/http_request/http_executor.py

```diff
@@ -338,6 +338,7 @@ def _format_template(
                     raise ValueError(f'Variable {variable_selector.variable} not found')
                 if escape_quotes and isinstance(variable, str):
                     value = variable.replace('"', '\\"')
+                    value = value.replace('\n', '\\n')
```

Comment:
Why do we manually escape chars?

How about 
```
if escape_quotes and isinstance(variable, str):
    value = json.dumps(variable)[1:-1]  # Remove the surrounding quotes added by json.dumps

```

Reply:
btw we should refactor more, because `escape_quotes ` is no longer the only thing we do here.

Reply:
Here is only variable characters, not yet to the body, so you can not use `json.dumps` function to parse it